src/META-INF/services/javax.xml.transform.TransformerFactory

```diff
@@ -1 +1 @@
-org.apache.xalan.xsltc.trax.TransformerFactoryImpl
+org.apache.xalan.processor.TransformerFactoryImpl
```

Comment:
Note this change, which I believe means a change in the default TransformerFactory for WF users. (I could be wrong.) The new value here is the one EAP has been using for many years. The existing value uses BCEL, which is not present in the xalan-j.jar that EAP ships.